ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -873,5 +873,68 @@
     "CaseEventID": "manageRepresentatives-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudgeGatekeeping",
```

Comment:
I really hate this duplication. Can we not use one event and * for pre and post state condition? i.e we know that hmcts admin / gatekeeper / judge can't see a case until it is in submitted. So, the event is not accessible in open state anyway. Would cut down your duplication a little

Reply:
I like this idea, we have AnyState.json and we can put the event in there, would really cut down the definitions if we're ok with using * (how does deleted state work? can they do * events?)

Reply:
LA is the only person who can see deleted state and they only have R permissions. So no events available even if they did have permissions to do this one

Reply:
I like this idea :)